targets/pna_nic/tests/.gitignore

```diff
@@ -0,0 +1,3 @@
+test*
+!test*.cpp
+!testdata
```

Comment:
most files in this PR, including this one, are missing the trailing newline character